src/ddspdrum/module.py

```diff
@@ -368,16 +368,21 @@ def _npyforward(self, mod_signal: np.array, phase: float = 0.0) -> np.ndarray:
 
         """
 
-        assert (mod_signal >= 0).all() and (mod_signal <= 1).all()
+        assert (mod_signal >= -1).all() and (mod_signal <= 1).all()
```

Comment:
Can you explain why we need this change? Is it only FmVCO where mod_signal >= -1 and for all the others its >= 0?

Reply:
Frequency modulation adjusts the frequency both above and below the central frequency; that's how the theory works and how you get the nice central pitch percept. 

But I'm thinking that mod's should be `-1 < mod < 1` in general though. Makes for a more flexable synth.

Reply:
Okay cool